content/docs/components/alert/theming.mdx

```diff
@@ -2,3 +2,186 @@
 id: alert
 scope: theming
 ---
+
+The `Alert` component is a multipart component. The styling needs to be applied
+to each part specifically.
+
+> To learn more about styling multipart components, visit the
+> [Component Style](/docs/styled-system/component-style#styling-multipart-components)
+> page.
+
```

Comment:
Can you put the `## Anatomy` section here with its parts? I'll prepare a custom illustration for you afterwards.